v2.1/import-data.md

```diff
@@ -29,53 +29,114 @@ $ cockroach sql --database=[database name] < statements.sql
 
-{{site.data.alerts.callout_success}}The <code>.sql</code> files generated by <code>pg_dump</code> provide better performance because they use the <code>COPY</code> statement instead of bulk <code>INSERT</code> statements.{{site.data.alerts.end}}
+<span class="version-tag">New in v2.1:</span> This section has instructions for getting data from Postgres dump files into CockroachDB using [`IMPORT`](import.html). It uses the [employees data set](https://github.com/datacharmer/test_db) that is also used in the [MySQL docs](https://dev.mysql.com/doc/employee/en/).  The data set was imported into Postgres from MySQL using [pgloader](https://pgloader.io).
 
-### Create PostgreSQL SQL file
+- [Step 1. Dump the Postgres database](#step-1-dump-the-postgres-database)
+- [Step 2. `IMPORT` the Postgres dump file](#step-2-import-the-postgres-dump-file)
 
-Which `pg_dump` command you want to use depends on whether you want to import your entire database or only specific tables:
+### Step 1. Dump the Postgres database
 
-- Entire database:
+To dump the 'employees' table from a Postgres database also named 'employees', run the [`pg_dump`](https://www.postgresql.org/docs/current/static/app-pgdump.html) command below.
 
-    {% include copy-clipboard.html %}
-    ~~~ shell
-    $ pg_dump [database] > [filename].sql
-    ~~~
+The following options are required:
 
-- Specific tables:
+- The `--inserts` option is required because the Postgres [`COPY`](https://www.postgresql.org/docs/current/static/sql-copy.html) statement is not yet supported.
 
-    {% include copy-clipboard.html %}
-    ~~~ shell
-    $ pg_dump -t [table] [table's schema] > [filename].sql
-    ~~~
+- The `--no-owner` option is required until we develop support for Postgres' object ownership statements.
 
-For more details, see PostgreSQL's documentation on [`pg_dump`](https://www.postgresql.org/docs/9.1/static/app-pgdump.html).
+- The `--no-privileges` option is required until we develop support for Postgres' access privilege statements.
 
-### Reformat SQL file
+{% include copy-clipboard.html %}
+~~~ shell
+$ pg_dump --inserts --no-privileges -t employees --no-owner employees > /tmp/employees.sql
+~~~
 
-After generating the `.sql` file, you need to perform a few editing steps before importing it:
+### Step 2. `IMPORT` the Postgres dump file
 
-1. Remove all statements from the file besides the `CREATE TABLE` and `COPY` statements.
-2. Manually add the table's [`PRIMARY KEY`](primary-key.html#syntax) constraint to the `CREATE TABLE` statement.
-  This has to be done manually because PostgreSQL attempts to add the primary key after creating the table, but CockroachDB requires the primary key be defined upon table creation.
-3. Review any other [constraints](constraints.html) to ensure they're properly listed on the table.
-4. Remove any [unsupported elements](sql-feature-support.html).
+To import the Postgres dump file, issue an `IMPORT` statement like the one below.
```

Comment:
This sentence is disconnected from the paragraph that follows it, and the `IMPORT` statement it's referring to is much later in the doc. Can we start this para with "You will need..." and add this sentence right before the `IMPORT` statement?

Reply:
Fixed.